ocp_resources/backup.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from ocp_resources.resource import NamespacedResource
+
+
+class Backup(NamespacedResource):
+    """
+    Backup object.
+    """
+
+    api_group = NamespacedResource.ApiGroup.VELERO_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        included_namespaces=None,
+        client=None,
+        teardown=False,
+        privileged_client=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        super().__init__(
+            name=name,
+            namespace=namespace,
+            client=client,
+            teardown=teardown,
+            privileged_client=privileged_client,
+            yaml_file=yaml_file,
+            **kwargs,
+        )
+        self.included_namespaces = included_namespaces
+
+    def to_dict(self):
+        res = super().to_dict()
+        if self.yaml_file:
+            return res
+        res.update({"spec": {"includedNamespaces": self.included_namespaces}})
```

Comment:
can `includedNamespaces` be `None` in spec?

Reply:
it can be None but the status will be failed in that case, should it still be an option?

Reply:
if this is mandatory to create a valid `Backup` resource then assert if not set